### PR TITLE
TCM-334 | Build dummy IAPSubscriptions screen

### PR DESCRIPTION
**Pull request by halilyesilyurt**
### Summary

- [x] Completed dummy iapsubscriptions screen.
- [x] Subscriptions's navigation changed from plans to new subscription screen.

### Issue

[TCM-334](https://hyfnla.atlassian.net/browse/TCM-334)

### Video/Screenshot

| With Subscription | Without Subscription|
|--------|------|
|subs2|subs|

### CODE REVIEW

**Comment by onodude** [comment]
please refer to the updated [design specs](https://www.figma.com/file/LXa0oBt9aSNOk0YMAeSM8Q/?node-id=3006%3A2)

**Comment by halilyesilyurt** [comment]
> please refer to the updated [design specs](https://www.figma.com/file/LXa0oBt9aSNOk0YMAeSM8Q/?node-id=3006%3A2)

Completed.

**Comment by halilyesilyurt** [comment]
updated pr images.

File: app/src/main/res/layout/fragment_subscriptions.xml [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+            android:id="@+id/lblTitle"
+            android:layout_width="0dp"
+            android:layout_height="60dp"
+            android:layout_marginStart="5dp"
+            android:fontFamily="@font/atcarquette_bold"
+            android:gravity="center_vertical"
+            android:padding="10dp"
+            android:text="@string/subscriptions_title"
+            android:textColor="@color/black"
+            android:textSize="12sp"
```
Comment (onodude) [inline app/src/main/res/layout/fragment_subscriptions.xml:46]: should 11sp

File: app/src/main/res/layout/fragment_subscriptions.xml [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+                    tools:text="10/10/2020" />
+
+            </com.trx.consumer.common.CommonView>
+
+        </com.trx.consumer.common.CommonView>
+
+        <com.trx.consumer.common.CommonButton
+            android:id="@+id/btnCancelSubscription"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
```
Comment (onodude) [inline app/src/main/res/layout/fragment_subscriptions.xml:161]: should have a height of 80dp and a top margin of 10dp

File: app/src/main/res/layout/fragment_subscriptions.xml [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+            android:layout_marginTop="40dp"
+            android:fontFamily="@font/atcarquette_bold"
+            android:gravity="center"
+            android:text="@string/subscriptions_cancel_subscription_label"
+            android:textAllCaps="true"
+            android:textColor="@color/red"
+            android:visibility="gone"
```
Comment (onodude) [inline app/src/main/res/layout/fragment_subscriptions.xml:168]: missing font size: 13sp

File: app/src/main/res/layout/fragment_subscriptions.xml [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+            <com.trx.consumer.common.CommonView
+                android:id="@+id/viewNextBill"
+                android:layout_width="match_parent"
+                android:layout_height="80dp"
+                android:layout_marginTop="10dp"
+                app:backgroundColor="@color/white"
+                app:layout_constraintTop_toBottomOf="@+id/viewLastBill">
+
+                <com.trx.consumer.common.CommonLabel
+                    android:id="@+id/lblNextBillDateLabel"
```
Comment (onodude) [inline app/src/main/res/layout/fragment_subscriptions.xml:125]: missing text color: greyDarkExtra

File: app/src/main/res/layout/fragment_subscriptions.xml [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+                android:id="@+id/viewLastBill"
+                android:layout_width="match_parent"
+                android:layout_height="80dp"
+                app:backgroundColor="@color/white"
+                app:layout_constraintTop_toTopOf="parent">
+
+                <com.trx.consumer.common.CommonLabel
+                    android:id="@+id/lblLastBillDateLabel"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
```
Comment (onodude) [inline app/src/main/res/layout/fragment_subscriptions.xml:86]: height should be 10dp

File: app/src/main/res/layout/fragment_subscriptions.xml [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+                    android:layout_marginStart="15dp"
+                    android:layout_marginTop="15dp"
+                    android:fontFamily="@font/atcarquette_medium"
```
Comment (onodude) [inline app/src/main/res/layout/fragment_subscriptions.xml:89]: should be bold

File: app/src/main/res/layout/fragment_subscriptions.xml [hunk 7]
```diff
@@ -0,0 +1,176 @@
...
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent"
+                    app:textAllCaps="true"
+                    tools:ignore="SmallSp" />
+
+
+                <com.trx.consumer.common.CommonLabel
+                    android:id="@+id/lblLastBillDate"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
```
Comment (onodude) [inline app/src/main/res/layout/fragment_subscriptions.xml:101]: height is 30dp

File: app/src/main/res/layout/fragment_subscriptions.xml [hunk 8]
```diff
@@ -0,0 +1,176 @@
...
+
+            <com.trx.consumer.common.CommonView
+                android:id="@+id/viewLastBill"
+                android:layout_width="match_parent"
+                android:layout_height="80dp"
+                app:backgroundColor="@color/white"
+                app:layout_constraintTop_toTopOf="parent">
+
+                <com.trx.consumer.common.CommonLabel
+                    android:id="@+id/lblLastBillDateLabel"
```
Comment (onodude) [inline app/src/main/res/layout/fragment_subscriptions.xml:84]: needs text color: grey

File: app/src/main/res/layout/row_subscriptions.xml [hunk 9]
```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.trx.consumer.common.CommonView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="190dp"
+    android:layout_marginBottom="15dp"
+    android:paddingStart="15dp"
+    android:paddingTop="10dp"
```
Comment (onodude) [inline app/src/main/res/layout/row_subscriptions.xml:9]: this doesn't seem to do anything?

File: app/src/main/res/layout/row_subscriptions.xml [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+    <com.trx.consumer.common.CommonLabel
+        android:id="@+id/lblTitle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="10dp"
+        android:fontFamily="@font/atcarquette_bold"
+        android:gravity="center_vertical"
+        android:textAllCaps="true"
+        android:textColor="@color/black"
+        android:textSize="12sp"
```
Comment (onodude) [inline app/src/main/res/layout/row_subscriptions.xml:22]: should be bold 11sp

File: app/src/main/res/layout/row_subscriptions.xml [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+    android:paddingStart="15dp"
+    android:paddingTop="10dp"
+    android:paddingEnd="15dp"
+    app:backgroundColor="@color/white">
+
+    <com.trx.consumer.common.CommonLabel
+        android:id="@+id/lblTitle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="10dp"
```
Comment (onodude) [inline app/src/main/res/layout/row_subscriptions.xml:17]: should be 45dp height and centered vertically

File: app/src/main/res/layout/row_subscriptions.xml [hunk 12]
```diff
@@ -0,0 +1,71 @@
...
+        android:textSize="12sp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="pay as you go" />
+
+    <com.trx.consumer.common.CommonLabel
+        android:id="@+id/lblDescription"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="15dp"
```
Comment (onodude) [inline app/src/main/res/layout/row_subscriptions.xml:31]: should be 30dp height and centered vertically

File: app/src/main/res/layout/row_subscriptions.xml [hunk 13]
```diff
@@ -0,0 +1,71 @@
...
+        android:textSize="14sp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/lblTitle"
+        tools:text="TRX On demand yearly" />
+
+    <com.trx.consumer.common.CommonLabel
+        android:id="@+id/lblPrice"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="10dp"
```
Comment (onodude) [inline app/src/main/res/layout/row_subscriptions.xml:44]: should be 30dp height and centered vertically

File: app/src/main/res/layout/fragment_subscriptions.xml (hunk 1 above)
Comment (halilyesilyurt) [reply app/src/main/res/layout/fragment_subscriptions.xml]: Completed.

File: app/src/main/res/layout/fragment_subscriptions.xml (hunk 2 above)
Comment (halilyesilyurt) [reply app/src/main/res/layout/fragment_subscriptions.xml]: Completed.

File: app/src/main/res/layout/fragment_subscriptions.xml (hunk 3 above)
Comment (halilyesilyurt) [reply app/src/main/res/layout/fragment_subscriptions.xml:175]: Completed.

File: app/src/main/res/layout/fragment_subscriptions.xml (hunk 4 above)
Comment (halilyesilyurt) [reply app/src/main/res/layout/fragment_subscriptions.xml:128]: Completed.

File: app/src/main/res/layout/fragment_subscriptions.xml (hunk 5 above)
Comment (halilyesilyurt) [reply app/src/main/res/layout/fragment_subscriptions.xml]: Completed.

File: app/src/main/res/layout/fragment_subscriptions.xml (hunk 6 above)
Comment (halilyesilyurt) [reply app/src/main/res/layout/fragment_subscriptions.xml]: Completed.

File: app/src/main/res/layout/fragment_subscriptions.xml (hunk 7 above)
Comment (halilyesilyurt) [reply app/src/main/res/layout/fragment_subscriptions.xml]: Completed.

File: app/src/main/res/layout/fragment_subscriptions.xml (hunk 8 above)
Comment (halilyesilyurt) [reply app/src/main/res/layout/fragment_subscriptions.xml:84]: Completed.

File: app/src/main/res/layout/row_subscriptions.xml (hunk 10 above)
Comment (halilyesilyurt) [reply app/src/main/res/layout/row_subscriptions.xml]: Completed.

File: app/src/main/res/layout/row_subscriptions.xml (hunk 11 above)
Comment (halilyesilyurt) [reply app/src/main/res/layout/row_subscriptions.xml]: Completed.

File: app/src/main/res/layout/row_subscriptions.xml (hunk 12 above)
Comment (halilyesilyurt) [reply app/src/main/res/layout/row_subscriptions.xml]: Completed.

File: app/src/main/res/layout/row_subscriptions.xml (hunk 13 above)
Comment (halilyesilyurt) [reply app/src/main/res/layout/row_subscriptions.xml]: Completed.

**Comment by halilyesilyurt** [comment]
updated pr images.

**Comment by onodude** [comment]
will be addressed after 
https://hyfnla.atlassian.net/browse/TCM-384

**Review by onodude — APPROVED** [review]

**Pull request merged**